.github/actions/setup-node/action.yaml

```diff
@@ -11,16 +11,16 @@ runs:
   using: "composite"
   steps:
     - name: Install pnpm
-      uses: pnpm/action-setup@v3
-      with:
-        version: 9.6
```

Comment:
removed to use the version from `packages.json`

Reply:
we specified `pnpm` version 9.7 in package.json and 9.6 here, which was conflicting.
cc: @BrunoQuaresma @aslilac